lib/rules/max-len.js

```diff
@@ -305,6 +322,11 @@ module.exports = {
                     // and step back by one
                     comment = comments[--commentsIndex];
 
+                    // check jsx comment
+                    if (isCommentInSingleLineJSXEmptyExpression(comment)) {
+                        comment = sourceCode.getNodeByRangeIndex(comment.range[0]).parent;
+                    }
```

Comment:
> This might execute relatively costly isCommentInSingleLineJSXEmptyExpression on some irrelevant comments like those inside code, but could end up being more optimal because the do-while loop in lines.forEach often gets back to the same comment, if I correctly understand the loop?

Thanks for the always careful review! I agree and, I'll care of it.